pkg/logs/input/kubernetes/launcher.go

```diff
@@ -218,6 +243,14 @@ func (l *Launcher) getAnnotation(pod *kubelet.Pod, container kubelet.ContainerSt
 	return ""
 }
 
+//getServiceLabel returns the standard service label for container if present
+func (l *Launcher) getServiceLabel(pod *kubelet.Pod, container kubelet.ContainerStatus) string {
```

Comment:
this doesn't need to be a method of thee `Launcher` struct

Reply:
done